tests/x11regressions/evolution/evolution_prepare_servers.pm

```diff
@@ -0,0 +1,76 @@
+# Evolution tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup dovecot and postfix servers as backend for evolution
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use strict;
+use base "x11regressiontest";
+use testapi;
+use utils;
+
+sub run() {
+    select_console('root-console');
+    pkcon_quit;
+
+    # add SLES repository and install dovecot
+    zypper_call("ar http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/ sle_server_repo");
```

Comment:
I was thinking about SLE15 when i was adding that line, but SLE15 is not out yet and I believe that adaptation for it will be easy. It works for 12SP3  (was tested, it use same base as 12SP2 ) , which is going to be live soon. 
The other way could be to use multimachine tests, but it looks like overkill just for this purpose.